### PR TITLE
feat(TCCUI-188) - show border on focus

### PR DESCRIPTION
**Pull request by ybaskaran**
**What is the problem this PR is trying to solve?**
The focus state border on Badge with dropdown needs to be improved to show the border on the whole Badge
https://jira.talendforge.org/browse/TCCUI-188

**What is the chosen solution to this problem?**
Add border on the whole badge and remove the outline from the dropdown element
http://2946.talend.surge.sh/components/?path=/story/navigation-badge--default

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [x] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

**Review by jsomsanith-tlnd — COMMENTED** [review]
Why do we have 2 different font-weight now ?

**Comment by ybaskaran** [comment]
> Why do we have 2 different font-weight now ?

I think it was set wrongly before to $font-weight-bold, all the category labels should have $font-weight-semi-bold as per the guidelines. Fixed now.
https://company-57688.frontify.com/document/92132#/ui-controls/badges-1597766704

File: packages/components/src/Badge/Badge.scss [hunk 1]
```diff
@@ -343,4 +348,14 @@ $tc-badge-disabled-opacity: 0.62;
 	&--value {
 		@include colored-badge(tint($lochmara, 70%), shade($lochmara, 30%));
 	}
+
+	&-dropdown {
+		.tc-badge-button {
+			&:not(.tc-badge-disabled) {
+				&:global(.ally-focus-within) {
+					border-color: rgba($lochmara, 1);
```
Comment (frassinier) [inline packages/components/src/Badge/Badge.scss:356]: why using rgba here?

Comment (ybaskaran) [reply packages/components/src/Badge/Badge.scss:356]: actually not required, but just using this to be in sync with the other badge types
https://github.com/Talend/ui/blob/4b3e2917ba645601781c8eac40e439adf7481973/packages/components/src/Badge/Badge.scss#L300

Comment (frassinier) [reply packages/components/src/Badge/Badge.scss:356]: no, please remove if you don't adjust alpha 🙏

Comment (ybaskaran) [reply packages/components/src/Badge/Badge.scss]: fixed

**Review by frassinier — APPROVED** [review]

**Review by jsomsanith-tlnd — APPROVED** [review]

**Pull request merged**